src/meta/assessment/originalDataPoint/odps/calc.ts

```diff
@@ -112,7 +112,7 @@ const calculatePrimaryForest = (props: { originalDataPoint: OriginalDataPoint })
     })
     const primaryForestPercent = Numbers.toBigNumber(originalDataPoint.values.primaryForestPercent)
 
-    return Numbers.mul(naturalForest, Numbers.div(primaryForestPercent, 100)).toNumber()
+    return Numbers.mul(naturalForest, Numbers.div(primaryForestPercent, 100))?.toNumber()
```

Comment:
This fixes a bug that we currently have in the main branch. I discovered it while testing for this PR.
If the natural regenerating forest percent is empty, the app crashes:

https://github.com/openforis/fra-platform/assets/41337901/1813c662-a871-4ee1-8545-e2e717c48ca8


Now:

https://github.com/openforis/fra-platform/assets/41337901/2b122b71-12f7-403f-b064-956fa4832320